src/main.ts

```diff
@@ -4,16 +4,18 @@ import piniaPluginPersistedState from "pinia-plugin-persistedstate";
 
 import Alert from "@/components/Alert.vue";
 import AuthLink from "@/components/AuthLink.vue";
+import VCalendar from "v-calendar";
 
 import App from "./App.vue";
 import router from "./router";
 
 import "./assets/css/style.scss";
 import "flowbite";
+import "v-calendar/dist/style.css";
 
 const app = createApp(App);
 
 app.component("Alert", Alert);
 app.component("AuthLink", AuthLink);
 
-app.use(router).use(createPinia().use(piniaPluginPersistedState)).mount("#app");
+app.use(router).use(createPinia().use(piniaPluginPersistedState)).use(VCalendar, {}).mount("#app");
```

Comment:
Don't set this up globally.. that creates unneeded bloat. Load it as a component where needed instead. https://vcalendar.io/vue-3.html#method-3-use-components-as-needed

Reply:
Fixed in [6a787f1]